lllm2/app.py

```
@@ -121,6 +125,7 @@ def start():
 def main():
     parser = argparse.ArgumentParser(description='lllm2 local LLM workbench')
     parser.add_argument('--port',type=int,default=8082)
+    parser.add_argument('--host',default='0.0.0.0',help='IPv4 bind address (default: all interfaces; use 127.0.0.1 for localhost only)')
```

Ruff (0.16.3)

[error] Possible binding to all interfaces

(S104)